### PR TITLE
main/links: upgrade to 2.19

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: main/links/APKBUILD [hunk 1]
```diff
@@ -1,25 +1,24 @@
...
-build () {
+build() {
 	cd "$builddir"
 	./configure \
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
 		--mandir=/usr/share/man \
-		--enable-javascript \
+		--disable-javascript \
```
Comment (5paceToast) [inline main/links/APKBUILD]: Why is javascript being changed to disabled?

**Comment by maxice8** [comment]
On Sun, Jun 09, 2019 at 09:48:08AM -0700, Chloe Kudryavtsev wrote:
> 5paceToast commented on this pull request.
> 
> 
> 
> >  	cd "$builddir"
>  	./configure \
>  		--build=$CBUILD \
>  		--host=$CHOST \
>  		--prefix=/usr \
>  		--mandir=/usr/share/man \
> -		--enable-javascript \
> +		--disable-javascript \
> 
> Why is javascript being changed to disabled?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8677#pullrequestreview-247406654

Fails to compile

**Comment by 5paceToast** [comment]
Okay, I've contacted the author (there's no easy bug report system).
Would recommend not merging until it's resolved (the error is specifically an upstream issue - removed header).

**Comment by maxice8** [comment]
On Sun, Jun 09, 2019 at 10:03:40AM -0700, Chloe Kudryavtsev wrote:
> Okay, I've contacted the author (there's no easy bug report system).
> Would recommend not merging until it's resolved (the error is specifically an upstream issue - removed header).
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8677#issuecomment-500228789

Ok, will wait.

**Comment by ncopa** [comment]
so as I understand, we want `--enable-javascript`, but it currently fails due tome missing header? Cant we just patch it?

**Comment by maxice8** [comment]
On Fri, Jun 28, 2019 at 08:05:32AM -0700, Natanael Copa wrote:
> so as I understand, we want `--enable-javascript`, but it currently fails due tome missing header? Cant we just patch it?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8677#issuecomment-506767683

No clue, i'm waiting on upstream

**Comment by 5paceToast** [comment]
@ncopa the missing header isn't a system header, but a header that used to be present in the upstream sources, but got removed.
It is therefore likely that APIs have been significantly changed.
I sent an email to upstream asking them what to do, but haven't heard back.
It's doubtable that upstream is dead, since the latest release is from end of March 2019, but we might consider disabling javascript since they (still, 19 days later) haven't responded.
Alternatively, we could try patching it, sure, but I'm doubtful of how well that'll go, considering the author themselves broke it.

**Pull request closed without merging**